test/modules/authorizer/extensions/AUT_EXT_VotingRoles_v1.t.sol

```diff
@@ -1007,10 +1015,21 @@ contract AUT_EXT_VotingRoles_v1Test is ModuleTest {
 
         vm.startPrank(address(_governor));
         for (uint i; i < users.length; ++i) {
+            uint before = _governor.threshold();
```

Comment:
I find the use of the word `governor` pretty confusing here. Made me really wonder why we get the threshold for the Voting Roles Authorizer from our Governor before realising that this is prob still a left over from before the `Governor` contract was written.

Maybe a good idea to do a quick rename here? Wdyt?

Reply:
yeah thats a good idea actually

Reply:
probably makes sense to do in a separate PR to `dev` though, just to keep this separate from the actual fix. will do that.